src/plugins/Mirror.cpp

```diff
@@ -154,17 +154,26 @@ void Mirror::readXML(XMLfileUnits& xmlconfig)
 	if(MT_MELAND_2004 == _type)
 	{
 		_melandParams.use_probability_factor = true;
-		_melandParams.velo_target = 0.4;
+		_melandParams.velo_target = 4.4;
```

Comment:
did you intend to change this value?

Reply:
Actually the best way would be to remove this line (and the above). The right value is assigned to the variable afterwards and if not the program is aborted anyway. See lines 160 - 166.